docs/notebooks/introduction.pct.py

```diff
@@ -243,3 +245,81 @@
     num_init=len(dataset.query_points),
     idx_best=arg_min_idx,
 )
+
+# %% [markdown]
+# # Try an alternative acquisition function
+
+# %% [markdown]
+# By default, Trieste uses Expected Improvement (EI) as its acqusition function when performing Bayesian optimization. However, many alternative acqusition functions have been developed. One such alternative is Max-value Entropy Search (MES), which approximates the distribution of current estimate of the global minimum and tries to decrease its entropy with each optimization step.
```

Comment:
i didn't realise you can put this all on one line, that's much easier than manually splitting the lines. I'll do that in future